OMCompiler/SimulationRuntime/c/simulation/solver/context.c

```diff
@@ -0,0 +1,84 @@
+/*
+ * This file is part of OpenModelica.
+ *
+ * Copyright (c) 1998-2022, Linköpings University,
+ * Department of Computer and Information Science,
+ * SE-58183 Linköping, Sweden.
+ *
+ * All rights reserved.
+ *
+ * THIS PROGRAM IS PROVIDED UNDER THE TERMS OF THIS OSMC PUBLIC
+ * LICENSE (OSMC-PL). ANY USE, REPRODUCTION OR DISTRIBUTION OF
+ * THIS PROGRAM CONSTITUTES RECIPIENT'S ACCEPTANCE OF THE OSMC
+ * PUBLIC LICENSE.
+ *
+ * The OpenModelica software and the Open Source Modelica
+ * Consortium (OSMC) Public License (OSMC-PL) are obtained
+ * from Linköpings University, either from the above address,
+ * from the URL: http://www.ida.liu.se/projects/OpenModelica
+ * and in the OpenModelica distribution.
+ *
+ * This program is distributed  WITHOUT ANY WARRANTY; without
+ * even the implied warranty of  MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE, EXCEPT AS EXPRESSLY SET FORTH
+ * IN THE BY RECIPIENT SELECTED SUBSIDIARY LICENSE CONDITIONS
+ * OF OSMC-PL.
+ *
+ * See the full OSMC Public License conditions for more details.
+ *
+ */
+
+#include "context.h"
+#include "../../simulation_data.h"
+
+const char *EVAL_CONTEXT_STRING[CONTEXT_MAX] = {
+  "context UNKNOWN",
+  "context ODE evaluation",
+  "context algebraic evaluation",
+  "context event search",
+  "context jacobian evaluation",
+  "context symbolica jacobian evaluation"
```

Comment:
We already have "context" in the print string so it would be "Set context context UNKNOWN"... :sweat_smile: 
```suggestion
  "UNKNOWN",
  "ODE evaluation",
  "algebraic evaluation",
  "event search",
  "jacobian evaluation",
  "symbolica jacobian evaluation"
```